pkg/registry/registry.go

```diff
@@ -6,7 +6,8 @@ import (
 )
 
 type v struct {
-	blobs blobs
+	blobs     blobs
+	manifests manifests
```

Comment:
What about `/v2/.../tags/list`?

Reply:
Listing tags isn't required
https://github.com/opencontainers/distribution-spec/blob/master/spec.md#listing-image-tags

Manifest has all the information though to do it.